diff --git a/node_modules/@xstate/react/dist/declarations/src/createActorContext.d.ts b/node_modules/@xstate/react/dist/declarations/src/createActorContext.d.ts
index 82fa40a..99d8d3a 100644
--- a/node_modules/@xstate/react/dist/declarations/src/createActorContext.d.ts
+++ b/node_modules/@xstate/react/dist/declarations/src/createActorContext.d.ts
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { ActorRefFrom, AnyStateMachine, AreAllImplementationsAssumedToBeProvided, SnapshotFrom, InternalMachineImplementations, InterpreterOptions, Observer, StateFrom } from 'xstate';
 export declare function createActorContext<TMachine extends AnyStateMachine>(machine: TMachine, interpreterOptions?: InterpreterOptions<TMachine>, observerOrListener?: Observer<StateFrom<TMachine>> | ((value: StateFrom<TMachine>) => void)): {
     useActor: () => [StateFrom<TMachine>, ActorRefFrom<TMachine>['send']];
-    useSelector: <T>(selector: (snapshot: SnapshotFrom<TMachine>) => T, compare?: (a: T, b: T) => boolean) => T;
+    useSelector: <T>(selector: (snapshot: FromState<TMachine>) => T, compare?: (a: T, b: T) => boolean) => T;
     useActorRef: () => ActorRefFrom<TMachine>;
     Provider: (props: {
         children: React.ReactNode;
